src/Contracts/DataModels.ts

```diff
@@ -496,3 +498,8 @@ export interface MemoryUsageInfo {
   freeKB: number;
   totalKB: number;
 }
+
+export interface ContainerConnectionInfo {
+  status: ConnectionStatusType;
+  //need to add ram and rom info
```

Comment:
we already get this from the memory call right?

Reply:
This is the initial allocation status check, will use existing memory call after connecting to network.

Reply:
we can add it in the next iteration